src/static/css/base.sass

```diff
@@ -35,6 +35,10 @@ h5
 p
   line-height: 1.65em
 
+li
+  line-height: 1.65em
+  margin-bottom: 11px
```

Comment:
Do we want all list items to have a bottom margin, or do we just want to set the spacing between list items?

Reply:
Actually I think between would be preferrable. But on the other hand, we don't bother with `<p>` either. Maybe we can fix it for both.

Reply:
Alright, we can live that for another PR 👍